code/modules/organs/wound.dm

```diff
@@ -276,7 +300,43 @@ datum/wound/cut/massive
 	stages = list("massive wound" = 70, "massive healing wound" = 50, "massive blood soaked clot" = 25, "massive angry scar" = 10,  "massive jagged scar" = 0)
 	damage_type = CUT
 
+/** PUNCTURES **/
+/datum/wound/puncture/can_worsen(damage_type, damage)
+	return 0
+/datum/wound/puncture/can_merge(var/datum/wound/other)
```

Comment:
You might also want to allow merging wounds. I'm not sure what you get by disallowing merging - it doesn't make wounds more severe - and you get nicer examine descriptions - e.g. a pair of X, many X, etc.